docs/shared/ungrib.yaml

```diff
@@ -0,0 +1,13 @@
+ungrib:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
```

Comment:
I'd suggest trying `cores: 1` instead of `nodes: 1` and keeping that if it works. Since `ungrib` is a serial (i.e. not parallel, not MPI, not OpenMPI) code, it only needs a single core to run. On some systems, users will have no choice but to run an `ungrib` job on a full node, but many HPCs (like NOAA RDHPCS) allow requesting that your job run on a node where other jobs might be running, too: On an n-core node, if you ask for 1 core, you get than and then 1/n of the node's memory. This can save both time (because your job may not queue as long) and money (on a, say, 96-core node, you'll pay only 1/96 what you would have paid for using the whole node!). Reflecting those facts in our example config might give users helpful information about what the codes requirements really are -- although there's more authoritative and comprehensive documentation for that elsewhere.